pkg/catalogv2/system/system_test.go

```diff
@@ -0,0 +1,36 @@
+package system
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+// TestGetIntervalOrDefault Tests that it will convert the input to a time.Duration or
+// returns a default value of 21600 seconds
+func TestGetIntervalOrDefault(t *testing.T) {
```

Comment:
A couple of tiny things about the test:
1. It's a good idea to start a test with `t.Parallel()`, unless it relies on global state.
2. Comments to test function names are a bit superfluous - the test name itself (or its subtests) should describe what behavior you're testing.